### PR TITLE
docs(README): update readme

### PR DESCRIPTION
**Pull request by ypahalajani**
Add description for starting the node server and updating the readme with respect to **export pdf** demo addition.

### CODE REVIEW

**Comment by ypahalajani** [comment]
@jigar1101 This PR will update the readme after resolving the #3 issue.

PLEASE DO NOT MERGE THIS, until #6 is merged.

**Review by jigar1101 — APPROVED** [review]
@ypahalajani Thanks for the update in Readme, it'll be helpful to other contributors.

**Pull request merged**
